.github/workflows/reusable_pip_test.yml

```diff
@@ -0,0 +1,35 @@
+on:
```

Comment:
You may want to rename this file to convey it's python 3.9 ?

Reply:
but this will require update file name everywhere on python bump

Reply:
Ok, sure. Can we do a 
```
with:
  python: '3.9'
```

Maybe then ?

Reply:
Or call it ".github/workflows/reusable_pip_test_minum_python.yml"